include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -135,6 +135,15 @@ class LAPACKFullMatrix : public TransposeTable<number>
   LAPACKFullMatrix<number> &
   operator/= (const number factor);
 
+  /**
+   * Set a particular entry of the matrix to a value.
+   * Thus, calling <code>A.set(1,2,3.141);</code> is entirely equivalent to the operation <code>A(1,2) = 3.141;</code>.
+   * This function exists for compatibility with the various sparse matrix objects.
+   */
+  void set (const size_type   i,
+            const size_type   j,
+            const number  value);
```

Comment:
Please fix the indentation of the parameters, both here and in the function definition.